interface/modules/custom_modules/oe-module-ehi-exporter/src/Services/EhiExporter.php

```diff
@@ -72,6 +72,50 @@ public function __construct(private $modulePublicDir, private $modulePublicUrl,
         $this->cryptoGen = new CryptoGen();
     }
 
```

Comment:
hi @adunsulag , just a minor php exception above
```
Deprecated: Creation of dynamic property OpenEMR\Modules\EhiExporter\Services\EhiExporter::$jobService is deprecated

Reply:
What PHP version are you running @stephenwaite?

Reply:
hi @sjpadgett, easy dev is on [8.2](https://php.watch/versions/8.2/dynamic-properties-deprecated)

Reply:
Thanks. I'm 8.1 guess I need to go ahead and upgrade to 8.3 for my windows dev but to be truthful, I'd rather not know about new errors!:)